### PR TITLE
'ipfs init': CLI + tests

### PR DESCRIPTION
**Pull request by noffle**
No HTTP route/tests, since initialization is pre-req of having an HTTP API available.

### CODE REVIEW

**Comment by noffle** [comment]
Right now `ipfs init` has no output to stdout. We could mimick go-ipfs' output, or do our own thing. Unless there's a pressing need for output though, I'm partial to following [the rule of silence](http://www.linfo.org/rule_of_silence.html).

File: src/cli/commands/init.js [hunk 1]
```diff
@@ -1,7 +1,11 @@
 const Command = require('ronin').Command
+const IpfsRepo = require('ipfs-repo')
+const Ipfs = require('../../core')
+const fsBlobStore = require('fs-blob-store')
+const utils = require('../utils')
 
 module.exports = Command.extend({
-  desc: 'Initialize ipfs local configuration',
+  desc: 'Initialize a local IPFS local node',
```
Comment (dignifiedquire) [inline src/cli/commands/init.js]: double local

File: tests/test-cli/test-init.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+      expect(err).to.not.exist
+      process.env.IPFS_PATH = oldRepoPath
+      done()
+    })
+  })
+
+  it('basic', (done) => {
+    nexpect.spawn('node', [process.cwd() + '/src/cli/bin.js', 'init'])
+      .run((err, stdout, exitcode) => {
+        expect(err).to.not.exist
```
Comment (dignifiedquire) [inline tests/test-cli/test-init.js]: Only ensuring that no error is thrown, seems a bit superficial, maybe check for directories/files to exist at least?

File: src/cli/commands/init.js (hunk 1 above)
Comment (noffle) [inline src/cli/commands/init.js]: Agghghgh thank you. <3

File: tests/test-cli/test-init.js (hunk 2 above)
Comment (noffle) [inline tests/test-cli/test-init.js]: Agreed. On it!

Comment (noffle) [inline tests/test-cli/test-init.js]: Done.

**Comment by noffle** [comment]
:horse: Ready to merge.

**Comment by diasdavid** [comment]
💥🌈🎉 thank you :D

**Pull request merged**